python3-pyslurm.spec

```diff
@@ -0,0 +1,51 @@
+Name:		python3-pyslurm
+Version:	20.02.0
+Release:	1%{?dist}
+Summary:	python3-pyslurm
+
+Group:		Development/Libraries
+License:	GPLv2
+URL:		https://github.com/PySlurm/pyslurm
+
+BuildRequires:	slurm-devel, python3, python3-devel, python3-pip
```

Comment:
Does slurm-devel need to be a specific version? Is it available in all repos across RHEL7/8, CentOS7/8 and the latest two versions of Fedora?

Reply:
The slurm-devel version should fit to the PySlurm version or better the api shoud fit together.
You could change this to...
BuildRequires: slurm-devel >= 20.02.0

Reply:
Could we remove this spec file from this PR and add it in after testing across multiple OSs? In the past, add some RPM and Deb support wasn't done uniformly and worked for some but often not for others.

Reply:
That is ok for me.